Limitations.md

```diff
@@ -1,5 +1,5 @@
 * [Overview](#overview)
-* [Definiton of a limitation](#definiton-of-a-limitation)
+* [Definition of a limitation](#definiton-of-a-limitation)
```

Comment:
Nice pickup. Could you also fix the actual heading this points to (line 48) as well? :-)
Might be nice if you split this into a separate little commit as well - fine to roll it into this PR, just have two commits in this PR. thx!